lib/cisco_node_utils/radius_server.rb

```diff
@@ -73,7 +144,8 @@ def ==(other)
     end
 
     def auth_port
-      config_get('radius_server', 'auth-port', @name)
+      platform == :ios_xr ? @auth_port : config_get('radius_server',
+                                                    'auth-port', ip: @name)
```

Comment:
No, acct_port (and auth_port) are provided in the constructor and are set as instance variables. Thus, the getter for these values are effectively the same as what :attr_reader does (creates methods that return instance variable values, that were originally set in the constructor)

The thing to understand here, is that for XR, it is a combination of ip, acct_port and auth_port that define a resources uniqueness.